testing-with-jest.test.js

```diff
@@ -8,20 +8,38 @@
 
 // --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
 
+describe("addThemUp", () => {
+  test("function called addThemUp that takes two numbers as arguments and returns the sum", () => {
+    expect(addThemUp(2, 5)).toEqual(7)
+  })
+})
 
 
 
 
 // --------------------1b) See the test fail. THEN write the code to make the test pass.
 
-
+const addThemUp = (num1, num2) => {
+  return num1 + num2
+}
 
 
 
 // --------------------2a) Create a test for a function called triangleArea that takes the base and the height of a triange and returns the area. The area of a triange is the base times the height divided by two.
 
-
+describe("triangleArea", () => {
+  test("function called triangleArea that takes the base and the height of a triange and returns the area. The area of a triange is the base times the height divided by two", () => {
+    expect(triangleArea(6, 6)).toEqual(18)
+  })
+})
 
 
 
 // --------------------2b) See the test fail. THEN write the code to make the test pass.
+
+const triangleArea = (num1, num2) => {
+return 6 * 6 / 2
+
+
+
+}
```

Comment:
Your spacing and indentation is a bit off here. Besides that, you're also hard-coding the return results, thus negating the purpose of the function. What if 12 and 12 was passed in to the function call? The arguments of the function are essentially useless at this point. 

Additionally, the arguments that should be passed in come from the expect, it's your function call.